src/coreclr/jit/block.cpp

```diff
@@ -1856,10 +1856,10 @@ weight_t BasicBlock::getCalledCount(Compiler* comp)
 //    compiler - Compiler instance
 //
 // Notes:
-//    with profie data: number of expected executions of this block, given
-//    one call to the method
+//    With profile data: number of expected executions of this block, given
+//    one call to the method.
 //
-weight_t BasicBlock::getBBWeight(Compiler* comp)
+weight_t BasicBlock::getBBWeight(Compiler* comp) const
```

Comment:
Thanks for the const cleanup. Quick question for my own learning: How liberally should I interpret the [guidelines](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/clr-jit-coding-conventions.md#1518-const) on const usage? Should I use it for short-lived locals (ex: `const bool removableJump = block->JumpsToNext() && ...`) in future changes?

Reply:
Using `const` on member functions is particularly useful, as the linked guidlines say. In addition, it does make sense to use it as frequently as possible in new code. For short-lived locals, it's a useful "documentation", especially in large functions, where a reader doesn't need to go searching for any modifications to the variable. Some cases like `const Type * const p;` sometimes seems too verbose, but if it helps understanding, go for it!